color_coding/functions.py

```diff
@@ -0,0 +1,30 @@
+from .constants import MAJOR_COLORS, MINOR_COLORS
```

Comment:
file name as functions.py gives a very generic meaning. It does not reflect the intent or content of the file.
The naming can be improved.

Reply:
I have made changes. Can you review it?

Reply:
Reviewed and no comments 👍